### DISCUSSION TITLE
ice_text.model文件的训练方式和数据集是什么

### DISCUSSION THREAD

**Discussion by oaifaye** [zai-org/ChatGLM3#786] [Q&A] [answered]
你好 我注意到chatglm使用的分词工具是sentencepiece加载ice_text.model，sentencepiece有多种训练方式包括UNIGRAM、BPE等，请问坐着使用的是什么方式呢，并且方便透露数据集吗？

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
相关技术细节没有公开
